doc/api/policy.md

```diff
@@ -196,4 +196,91 @@ module.exports = function fn(...args) {
 };
 ```
 
+### Scopes
+
+To set configuration for many resources at once you can use the `"scopes"`
+field of a manifest. `"scopes"` works by matching resources by their segments.
+If a dependency is not found, the resource may include `"cascade": true` which
+will delegate to a scope. Scopes may also use `"cascade": true`. The scope for
+cascading is found by recursively reducing the resource URL by removing segments
+for [special schemes][], keeping trailing `"/"` suffixes and removing the query
+and hash fragment. This leads to the eventual reduction of the URL to its
+origin. If the URL is non-special the scope will be located by the URL's origin.
+If no scope is found for the origin or in the case of opaque origins, a protocol
+string can be used as a scope.
+
+Note, `blob:` URLs adopt their origin from the path they contain, and so a scope
+of `"blob:https://nodejs.org/"` will have no effect since no URL can have that
+origin and it will use `https:` for its protocol scope. For opaque origin
```

Comment:
Can have which origin? It's not clear here

Reply:
Specifically, it's probably safer to assume that most readers will not have enough of an understanding of "origin" and "opaque origin" to make sense of this

Reply:
can have `blob:https://nodejs.org/` as an origin.

Reply:
fixed